composer.json

```diff
@@ -14,16 +14,15 @@
         "drupal-composer/drupal-scaffold": "~2.2",
         "drupal/config_devel": "~1.2",
         "drupal/console": "~1.0",
-        "drupal/drupal-extension": "~4.0",
+        "drupal/drupal-extension": "~4.0.0@beta",
```

Comment:
Why the change in behat extention? 

https://github.com/openeuropa/drupal-module-template/blob/master/composer.json#L17